src/Data/ProgrammesDb/Entity/RefOptions.php

```diff
@@ -3,14 +3,18 @@
 namespace BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Exception\InvalidArgumentException;
 use Gedmo\Timestampable\Traits\TimestampableEntity;
-use InvalidArgumentException;
 
 /**
  * @ORM\Entity()
+ * @ORM\Table(indexes={@ORM\Index(name="ref_options_idx", columns={"guid", "entity_id"})})
```

Comment:
We need to be able to look things up based on entity id only, so this needs to be two indexs rather than one index with two columns.

If you add `unique=true` to the guid column definition doctrine will add an index for it, so you'd only need to specify the index for the entity_id.

Also check https://github.com/bbc/programmes-pages-service/blob/master/src/Data/ProgrammesDb/Entity/PipsChange.php for how we format this annotation, so it is easy to add new indexs if we ever need to